internal/dcache/rpc/client/functions.go

```diff
@@ -391,6 +420,13 @@ func PutChunk(ctx context.Context, targetNodeID string, req *models.PutChunkRequ
 
 			// Fall through to release the RPC client.
 			resp = nil
+		} else {
+			//
+			// The RPC call to the target node succeeded. If the node or the RV is marked negative or iffy,
+			// clear it now.
+			//
+			cp.removeNegativeNode(targetNodeID)
+			cp.removeIffyRvId(req.Chunk.Address.RvID)
```

Comment:
General question:
Btw, I'm curious, when is this actually helping.
If the node was negative then getRPCClient() would have failed.
So this will only help for the rare case where one of the other threads marked it negative after this thread got its client and made the RPC calls. Right?